src/guide/1/geofencing/go/index.md

```diff
@@ -130,7 +130,7 @@ We have now programmed the subscription side of the test.
 
 Now let's move on to the publish side of the test. Here we will create a document that represents the user's location, placed inside the circular boundary around Big Ben.
 
-We will program the *Publish* function so that it creates a document that contains three fields: `firstName`, `lastName` and `location`.
+We will program the *Publish* function so that it creates a document containings three fields: `firstName`, `lastName` and `location`.
```

Comment:
```suggestion
We will program the *Publish* function so that it creates a document containing three fields: `firstName`, `lastName` and `location`.
```